Offices/Offices.Data/Extensions/RepositoriesExtension.cs

```diff
@@ -13,7 +14,7 @@ public static class RepositoriesExtension
         public static IServiceCollection AddRepositories(this IServiceCollection services)
         {
             // Adding repositories in container
-            services.AddScoped(typeof(IGenericRepository<>), typeof(GenericRepository<>));
+            services.AddScoped<IGenericRepository<BaseEntity>, GenericRepository<BaseEntity>>();
```

Comment:
Not sure that this will not allow you to use it as generic. Previous version was better

Reply:
Fixed